src/customFileParser/loadCompressedGeometryData.ts

```diff
@@ -1,24 +1,24 @@
 import CustomFileReader from './CustomFileReader';
 import { IdToFileGeometryName } from './parserParameters';
-import { GeometryIndexHandler } from './sharedFileParserTypes';
+import { CompressedGeometryData } from './sharedFileParserTypes';
 
 export default function loadGeometryIndexes(file: CustomFileReader, sectorEndLocation: number):
-    GeometryIndexHandler[] {
+    CompressedGeometryData[] {
 
-  const geometryIndexes: GeometryIndexHandler[] = [];
+  const geometryIndexes: CompressedGeometryData[] = [];
   while (file.location < sectorEndLocation) {
-    const name = IdToFileGeometryName[file.readUint8()];
-    const geometryCount = file.readUint32();
+    const type = IdToFileGeometryName[file.readUint8()];
+    const count = file.readUint32();
     const attributeCount = file.readUint8();
     const byteCount = file.readUint32();
-    const nodeIds = file.getNodeIdReader(geometryCount);
-    const indexes = file.getFibonacciDecoder(byteCount);
+    const nodeIds = file.getNodeIdReader(count);
+    const indexes = file.getFibonacciDecoder(byteCount, count * attributeCount);
 
-    const newGeometry: GeometryIndexHandler = {
-      name: name,
+    const newGeometry: CompressedGeometryData = {
+      type: type,
```

Comment:
Doesn't the linter want you to just do `type,` when you have the same name `type: type`?